app/src/main/res/values/donottranslate.xml

```diff
@@ -12,8 +12,10 @@
     <item name="customize_fonts" type="string" translatable="false">customize_fonts</item>
     <item name="font_sizes" type="string" translatable="false">font_sizes</item>
     <item name="dhis_url" type="string" translatable="false">dhis_url</item>
+    <item name="dhis_user" type="string" translatable="false">dhis_user</item>
+    <item name="dhis_password" type="string" translatable="false">dhis_password</item>
     <item name="org_unit" type="string" translatable="false">org_unit</item>
-
+    <item name="pull_metadata" type="string" translatable="false">pull_metadata</item>
```

Comment:
@Strabagkantza You added this string to the donottranslate file but you never use it. Why? BTW, remember you're here adding it to the main branch, and not to the myanmar variant...do you need it to be here?